.github/workflows/create-release.yml

```
@@ -0,0 +1,52 @@
+name: Create Release
+
+on:
+  push:
+    tags:
+      - 'v*' # Trigger on tags with prefix 'v'
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4.1.1
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)